compiler/rustc_codegen_gcc/src/builder.rs

```diff
@@ -1425,6 +1428,23 @@ impl<'a, 'gcc, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'gcc, 'tcx> {
     fn instrprof_increment(&mut self, _fn_name: RValue<'gcc>, _hash: RValue<'gcc>, _num_counters: RValue<'gcc>, _index: RValue<'gcc>) {
         unimplemented!();
     }
+
+    fn from_const_alloc(&mut self, layout: TyAndLayout<'tcx>, alloc: ConstAllocation<'tcx>, offset: Size) -> OperandRef<'tcx, RValue<'gcc>> {
+        assert_eq!(alloc.inner().align, layout.align.abi);
+        let ty = self.type_ptr_to(layout.gcc_type(self));
+        if layout.size == Size::ZERO {
+            OperandRef::new_zst(self, layout)
+        }
+        else {
+            let init = const_alloc_to_gcc(self, alloc);
+            let base_addr = self.static_addr_of(init, alloc.inner().align, None);
+
+            let array = self.const_bitcast(base_addr, self.type_i8p());
+            let value = self.context.new_array_access(None, array, self.const_usize(offset.bytes())).get_address(None);
+            let value = self.const_bitcast(value, ty);
+            self.load_operand(PlaceRef::new_sized(value, layout))
+        }
+    }
```

Comment:
codegen_gcc is updated. Why isn't codegen_cranelift? It doesn't use that trait?

Reply:
It doesn't.